docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3528,8 +3537,9 @@ function viewParticipants() {
     const btnDiv = document.createElement('div');
 
     btnDiv.classList.add('btn-group');
-
-    btnDiv.appendChild(createButton('Mute', muteMember));
+    const muteBtn = createButton('Mute', muteMember)
```

Comment:
minor thing, but it would be nicer to just modify createButton to take an extra parameter like "options" or "props" that would then set these props on the new button so we could just call it like this: 
`btnDiv.appendChild(createButton('Mute', muteMember, {id: 'mute-participant-btn'}));`

Reply:
taken care